nixos/modules/services/networking/wstunnel.nix

```diff
@@ -232,13 +239,6 @@ let
           default = true;
         };
 
-        # The original argument name `websocketPingFrequency` is a misnomer, as the frequency is the inverse of the interval.
-        websocketPingInterval = lib.mkOption {
```

Comment:
Changes look good. Maybe you could add a `mkRenamedOption` to avoid surprises?

Reply:
It's not needed, the option for the client doesn't change, we merge `commonOptions` into the options for both client and server, so moving the option there just makes it available for the server as well as for the client.